crates/wasmtime/src/config.rs

```diff
@@ -109,6 +109,8 @@ pub struct Config {
     pub(crate) memory_init_cow: bool,
     pub(crate) memory_guaranteed_dense_image_size: u64,
     pub(crate) force_memory_init_memfd: bool,
+    #[cfg(feature = "async")]
+    pub(crate) reset_stack_page: bool,
```

Comment:
This can't be set by the user.

Reply:
What do you think would be a better way?

Reply:
Adding a setter like https://github.com/bytecodealliance/wasmtime/blob/d1fb2405d769b3eb81873d2a77b6b75e8d931936/crates/wasmtime/src/config.rs#L598-L603